docs/xdr/features/collect/integrations/endpoint/trend_micro_deep_security.md

```diff
@@ -0,0 +1,52 @@
+uuid: b2d961ae-0f7e-400b-879a-f97be24cc02d
+name: Trend Micro Deep Security / Workload Security
+type: intake
+
+## Overview
+
+Trend Micro Deep Security / Workload Security detect malware, vulnerabilities and any other threats.
```

Comment:
```suggestion
Trend Micro Deep Security / Workload Security detect malware, vulnerabilities and some other threats.
```

*any other threats* seems very magical to me.

Maybe remove this part or be more specific ?